src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepAnnotationFileWriter.java

```diff
@@ -32,18 +33,22 @@
 
     private final VepProcess vepProcess;
 
-    public VepAnnotationFileWriter(Long timeoutInSeconds, Integer chunkSize, AnnotationParameters annotationParameters) {
+    public VepAnnotationFileWriter(AnnotationParameters annotationParameters, Integer chunkSize, Long timeoutInSeconds) {
```

Comment:
mmh, after changing this, I saw that making it primitive, becomes required in other steps like annotation load, or annotation metadata.

We are checking that the parameter is provided in the generateAnnotationStep parameters validator, so I would leave it as Long.